### ISSUE TITLE
How much parallel data?

### ISSUE THREAD

**Issue by zidsi** [fe1ixxu/ALMA#27]
Figure 4 of your (first) paper shows degradation if "too much" training data is used. It suggests 10k and 100k as sweetpoint for 7B model. 

Was the same observed with 13B model and was the same regardless of used full vs. LORA SFT?

Also wondering if this could be "spoted" in loss curve while training or (expensive) checkpoint evaluation should be performed?

And last but not least - this 10k/100k guidence goes for total samples count or "per language pair"?

**Comment by fe1ixxu** [comment]
Thanks for your interest and sorry about the delayed response! I was just back online from the Spring Festival.

>Was the same observed with 13B model and was the same regardless of used full vs. LORA SFT?

The observation only applies to full SFT. LoRA SFT needs a further investigation.

>Also wondering if this could be "spoted" in loss curve while training or (expensive) checkpoint evaluation should be performed?

 I don't remember there are some spotted points in the loss curve. All checkpoints are re-trained from scratch when using different size of data, and picked with the one having lowest eval loss.

>And last but not least - this 10k/100k guidence goes for total samples count or "per language pair"?

It is per language pair.

Let me know if you have further questions!
